static/app/utils/url/stripURLOrigin.tsx

```diff
@@ -0,0 +1,16 @@
+import {safeURL} from 'sentry/utils/url/safeURL';
+
+/**
+ * Accept a url like:
+ * `https://example.com/path/name?query=params#hash` and return
+ * `/path/name?query=params#hash`
+ */
+export default function stripURLOrigin(url: string): string {
```

Comment:
@ryan953 I renamed this to include URL in the naming and better reflect that it operates on a URL like string